fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLauncherMonitor.java

```diff
@@ -103,7 +114,7 @@ public void run() {
             try {
                 outReader = new BufferedReader(new InputStreamReader(process.getInputStream()));
                 while (!isStop && (line = outReader.readLine()) != null) {
-                    LOG.info("monitor log: " + line);
+                    outputBuffer.append(line + '\n');
```

Comment:
why not just output to the `outputStream`? So that we don't need to buffer a log of content in memory?

Reply:
ok. I changed it